containers/session-containers/skaha-desktop/Dockerfile.centos.xfce.vnc

```diff
@@ -9,7 +9,8 @@ LABEL io.k8s.description="Headless VNC Container with Xfce window manager, firef
       io.openshift.non-scalable=true
 
 # BM: Install fonts so remote displays work
-RUN yum update -y && yum install -y xorg-x11-fonts* gedit rsync gimp gimp-gmic
+RUN yum update -y && yum install -y xorg-x11-fonts* gedit rsync gimp gimp-gmic gnome-terminal epel-release 
+RUN yum update -y && yum install -y jq
```

Comment:
Should jq just be added to the update list on the line above (line 12)?

Reply:
No, jq required epel-release to have been installed first.